### ISSUE TITLE
Are the information buttons needed?

### ISSUE THREAD

**Issue by lognaturel** [grzesiek2010/collectTester#6]
@grzesiek2010 The button names are clear enough that I'm not really feeling like the information buttons add much.

What about including the code instead? I mean something like
```
Intent i = new Intent(Intent.ACTION_VIEW);
i.setType("vnd.android.cursor.dir/vnd.odk.form");
```

That seems it would be more useful! @yanokwa what do you think?

**Comment by grzesiek2010** [comment]
@lognaturel Do you want to remove these buttons at all or change messages using code? because I don't understand. It's all the same to me so you decide what's better.

**Comment by lognaturel** [comment]
I mean leave the buttons but put the code in the dialogs. That way it's really useful for a developer who wants to connect what they see in the tester app with what they'll do in their own app. How does that sound?

**Comment by grzesiek2010** [comment]
@lognaturel 

> How does that sound?
I'm not sure what is the best for use. I added [fragments of code](https://github.com/grzesiek2010/collectTester/pull/7) as you proposed but only for 4 buttons since I can't do the same for FORM LIST and INSTANCES LIST because these buttons opens my own activity so it doesn't make sense to display any code there. That's why I kept original messages too.

Let me know what you think.
